Examples/Io/Performance/ActsExamples/Io/Performance/VertexPerformanceWriter.cpp

```diff
@@ -434,21 +436,22 @@ ActsExamples::ProcessCode ActsExamples::VertexPerformanceWriter::writeT(
       for (const auto& particle : allTruthParticles) {
         SimBarcode vtxId =
             particle.particleId().setParticle(0).setSubParticle(0);
-        contributingTruthVertices.push_back(vtxId);
+        contributingTruthVertices.emplace_back(vtxId, 1);
       }
     }
 
     // Find true vertex that contributes most to the reconstructed vertex
-    std::map<SimBarcode, int> fmap;
-    for (const SimBarcode& vtxId : contributingTruthVertices) {
-      fmap[vtxId]++;
+    std::map<SimBarcode, std::pair<int, double>> fmap;
+    for (const auto& [vtxId, weight] : contributingTruthVertices) {
+      ++fmap[vtxId].first;
+      fmap[vtxId].second += weight;
     }
-    int maxOccurrence = -1;
+    double maxOccurrence = -1;
```

Comment:
Consider using lowest double value instead of -1.

Reply:
reverted that to an int. it has the same behavior as before